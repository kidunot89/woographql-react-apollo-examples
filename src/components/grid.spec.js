import { render } from '@testing-library/react';

import Grid from './grid';

describe('Grid component', () => {
  const { baseElement } = render(<Grid />);

  it('renders successfully', () => {
    expect(baseElement).toMatchSnapshot();
  });
});
