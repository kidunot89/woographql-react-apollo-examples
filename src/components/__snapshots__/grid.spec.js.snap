// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid component renders successfully 1`] = `
<body>
  <div>
    <div
      class="sc-bczRLJ hmnXFb"
    />
  </div>
</body>
`;
